bin/console.php

```diff
@@ -1,15 +1,15 @@
 #! /usr/bin/env php
 <?php
 
-use Acme\controllerCommand;
+use TrendingRepos\Command\CreateControllerCommand;
 use Symfony\Component\Console\Application;
 
 
 require 'vendor/autoload.php';
 
 $app = new Application("Project is under development", "1.0");
 
-$app->add(new controllerCommand(new GuzzleHttp\Client()));
+$app->add(new CreateControllerCommand(new GuzzleHttp\Client()));
 
 
 
```

Comment:
@vladimino i didn't see that before pushing to github, do you use the desktop version of github to view changes before pushing ?

Reply:
> do you use the desktop version of github

No, I don't. What editor do you use? It's quite obvious in PhpStorm, for example:

<img width="588" alt="Screenshot 2019-11-17 09 57 27" src="https://user-images.githubusercontent.com/63159/69005314-c3fd9200-0920-11ea-8a43-39df5300d572.png">

And this is how it can look:

<img width="598" alt="Screenshot 2019-11-17 09 58 01" src="https://user-images.githubusercontent.com/63159/69005317-cf50bd80-0920-11ea-9e04-58d26ab84bea.png">

At least 4 lines less!